src/EnergyPlus/DElightManagerF.cc

```diff
@@ -365,8 +365,7 @@ namespace DElightManagerF {
                                       surf.Sides);
 
                                 // Write out the vertex coordinates for each vertex
-                                int const iNumVertices = surf.Sides; // Counter for surface vertices
-                                for (int ivert = 1; ivert <= iNumVertices; ++ivert) {
+                                for (int ivert = 1; ivert <= surf.Sides; ++ivert) {
```

Comment:
This was a good one. Since this int wasn't really used for anything I replaced all 3.

[src/EnergyPlus/DElightManagerF.cc:476]:(style),[shadowVariable],Local variable 'iNumVertices' shadows outer variable
[src/EnergyPlus/DElightManagerF.cc:536]:(style),[shadowVariable],Local variable 'iNumVertices' shadows outer variable